Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    public Rigidbody2D body;
+    public SpriteRenderer sprite;
+    public Animator animator;
+    public float speed = 8.0f;
+
+    Vector3 moveDirection = Vector3.zero;
+    string lastAnimation = "";
+
+
+    void Start() 
+    {
+        SetAnimation("idle_down");
+    }
+
+    void Update()
+    {
+        UpdateAnimator();
+    }
+
+    void FixedUpdate()
+    {
+        moveDirection = new Vector3(
+            Input.GetAxisRaw("Horizontal"), 
+            Input.GetAxisRaw("Vertical"), 
+            0.0f
+        );
+        moveDirection.Normalize();
+        
+        body.velocity = moveDirection * speed * Time.fixedDeltaTime;
```

Comment:
Ive never done it like this but [in this link](https://gamedev.stackexchange.com/questions/113178/when-should-i-use-velocity-versus-addforce-when-dealing-with-player-objects) Im reading that changing velocity of an objet doesnt consider anymore other forces applied to this object at that frame. So i dont know if thats what you want but you might consider using AddForce instead. Sorry for the syntax but im not using my keyboard

Reply:
Right, I struggled a lot to find a way of making the controls feels right and i may have used that as debug, i ll try again with AddForce to see if it feels any different

Reply:
You can use AddForce(force, ForceMode.Impulse), I think its more what you want here

Reply:
Tested several method, playing with player acceleration instead of controlling directly the velocity gives acceleration/deceleration behavior, it adds realism but remove some fine control tunning. And I think that this kind of game requires more fine tuning than realism